drizzlepac/hlautils/catalog_utils.py

```diff
@@ -614,48 +638,27 @@ def measure_sources(self, filter_name):
         flag_col = Column(name="Flags", data=np.zeros_like(photometry_tbl['ID']), dtype=np.int64)
         photometry_tbl.add_column(flag_col)
 
-        # Add null-value "TotMag(<outer radiiArc>)" and "TotMag(<outer radiiArc>)" columns
-        empty_tot_mag = MaskedColumn(name="TotMag({})".format(self.aper_radius_arcsec[1]), fill_value=None, mask=True,
-                                     length=len(photometry_tbl["XCENTER"].data), dtype=np.int64)
-        empty_tot_mag_err = MaskedColumn(name="TotMagErr({})".format(self.aper_radius_arcsec[1]), fill_value=None, mask=True,
-                                         length=len(photometry_tbl["XCENTER"].data), dtype=np.int64)
-        photometry_tbl.add_column(empty_tot_mag)
-        photometry_tbl.add_column(empty_tot_mag_err)
-
         # build final output table
-        final_col_order = ["XCENTER", "YCENTER", "RA", "DEC", "ID", "MAG_{}".format(self.aper_radius_arcsec[0]),
-                           "MAG_{}".format(self.aper_radius_arcsec[1]), "MERR_{}".format(self.aper_radius_arcsec[0]),
-                           "MERR_{}".format(self.aper_radius_arcsec[1]), "MSKY", "STDEV",
-                           "FLUX_{}".format(self.aper_radius_arcsec[1]), "TotMag({})".format(self.aper_radius_arcsec[1]),
-                           "TotMagErr({})".format(self.aper_radius_arcsec[1]), "CI", "Flags"]
+        final_col_order = ["XCENTER", "YCENTER", "RA", "DEC", "ID", "MagAp1", "MagErrAp1", "MagAp2", "MagErrAp2",
+                           "MSkyAp2", "StdevAp2", "FluxAp2", "CI", "Flags"]
         output_photometry_table = photometry_tbl[final_col_order]
 
         # format output table columns
-        final_col_format = {"RA": "13.10f", "DEC": "13.10f", "MAG_{}".format(self.aper_radius_arcsec[0]): '6.3f',
-                            "MAG_{}".format(self.aper_radius_arcsec[1]): '6.3f',
-                            "MERR_{}".format(self.aper_radius_arcsec[0]): '6.3f',
-                            "MERR_{}".format(self.aper_radius_arcsec[1]): '6.3f', "MSKY": '10.8f', "STDEV": '10.8f',
-                            "FLUX_{}".format(self.aper_radius_arcsec[1]): '10.8f', "CI": "7.3f"}
+        final_col_format = {"RA": "13.10f", "DEC": "13.10f", "MagAp1": '6.3f', "MagErrAp1": '6.3f', "MagAp2": '6.3f',
+                            "MagErrAp2": '6.3f', "MSkyAp2": '10.8f', "StdevAp2": '10.8f',
+                            "FluxAp2": '10.8f', "CI": "7.3f"}
```

Comment:
The current formatting is fine, but we could definitely get away with only '10.4f' instead of '10.8f' for Stdev columns.

Reply:
issue resolved in [51f079f](https://github.com/spacetelescope/drizzlepac/pull/470/commits/51f079fec7656d2d06d40f975460c89a0fda4a78)